src/main/backend.ts

```diff
@@ -41,6 +41,26 @@ export async function isAppReady(): Promise<boolean> {
   );
 }
 
+/** Gets a Scope object with user, tags and extra */
+function getScope(): Partial<ScopeContext> | undefined {
```

Comment:
Is there a better way than this to get an object with `user`, `tags` and `extra` from the scope?

Reply:
Not really, although I'd still rewrite it as:

```js
function getScope(): Partial<ScopeContext> {
  const scope = getCurrentHub().getScope() as any;

  if (!scope) {
    return {};
  }

  /* eslint-disable @typescript-eslint/no-unsafe-member-access */
  return {
    ...(scope._user && { user: scope._user }),
    ...(scope._tags && { tags: scope._tags }),
    ...(scope._extra && { extra: scope._extra }),
  };
}
```

It should be totally safe to always set initial scope to `{}` on line 220 then as well.